src/HDInsight/HDInsight/help/Disable-AzHDInsightMonitoring.md

```diff
@@ -1,30 +1,30 @@
-﻿---
+---
 external help file: Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll-Help.xml
 Module Name: Az.HDInsight
-online version: https://docs.microsoft.com/en-us/powershell/module/az.hdinsight/disable-azhdinsightoperationsmanagementsuite
+online version: https://docs.microsoft.com/en-us/powershell/module/az.hdinsight/disable-azhdinsightmonitoring
 schema: 2.0.0
 ---
 
-# Disable-AzHDInsightOperationsManagementSuite
+# Disable-AzHDInsightMonitoring
 
 ## SYNOPSIS
-Disables Operations Management Suite (OMS) in a HDInsight cluster and relevant logs will stop flowing to the OMS workspace specified during enable.
+Disables monitoring in a HDInsight cluster and relevant logs will stop flowing to the monitoring workspace specified during enable.
 
 ## SYNTAX
 
 ```
-Disable-AzHDInsightOperationsManagementSuite [-Name] <String> [-ResourceGroupName <String>]
+Disable-AzHDInsightMonitoring [-Name] <String> [-ResourceGroupName <String>]
  [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Disable-AzHDInsightOperationsManagementSuite** cmdlet disables Operations Management Suite (OMS) in a Azure HDInsight cluster.
+The **Disable-AzHDInsightMonitoring** cmdlet disables monitoring in a Azure HDInsight cluster.
 
 ## EXAMPLES
 
 ### Example 1
 ```
-PS C:\> Disable-AzHDInsightOMS -Name testcluster
+PS C:\> Disable-AzHDInsightMonitoring -Name testcluster
 
 ErrorInfo  :
```

Comment:
@aim-for-better the output for these examples need to be updated since this cmdlet now returns a `bool`

Also, make sure that the output type is updated at the bottom of this file by running `Update-MarkdownHelpModule` with the `-UpdateInputOutput` parameter

Reply:
> @aim-for-better the output for these examples need to be updated since this cmdlet now returns a `bool`
> 
> Also, make sure that the output type is updated at the bottom of this file by running `Update-MarkdownHelpModule` with the `-UpdateInputOutput` parameter

updated.